modules/UI/videolayout/FilmStrip.js

```diff
@@ -12,6 +12,46 @@ const FilmStrip = {
     init (eventEmitter) {
         this.filmStrip = $('#remoteVideos');
         this.eventEmitter = eventEmitter;
+        this.filmStripIsVisible = true;
+        this.renderFilmstripToolbar();
+        this.activateHideButton();
+    },
+
+    /**
+     * Attach 'click' listener to "hide filmstrip" button
+     */
+    activateHideButton () {
```

Comment:
We tend to call those kind of functions registerListeners in other classes, so if we want to be consistent we should rename this one. Also adding a _ to the name will better indicate that it's a private function.